### ISSUE TITLE
Documentation – Pattern for form-field-text

### ISSUE THREAD

**Issue by AlexanderBoenninger** [arnoson/kirby-forms#5]
The Placeholder says [a-b]+

A use case would be nice, right?

**Comment by arnoson** [comment]
Yes maybe having a `user` usecase and change the placeholder to `\w{3,16}` like in the [MDN example](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern). But where to document such a usecase, in the readme or as an info next to the placeholder?

**Comment by arnoson** [comment]
In the end the pattern field ist directly added to the corresponding html input field, nothing special is added by this plugin. So maybe a link to the MDN documentation would already be enough
